lfs_util.h

```diff
@@ -43,6 +43,10 @@ extern "C"
 {
 #endif
 
+// Enables thread-safe wrappers using the lock/unlock callbacks in lfs_config
+#ifndef LFS_THREAD_SAFE
```

Comment:
Very minor, but I'm of the opinion we should change this to LFS_THREADSAFE (no space) to match the incoming LFS_READONLY (https://github.com/littlefs-project/littlefs/pull/480)

Reply:
fixed